.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+language: python
+python:
+  - 3.3
+  - 3.4
+  - 3.5
+  - pypy3
+install:
+  - pip install flake8
+script:
+  - flake8 --ignore E501 homu
```

Comment:
For now at least, I've disabled warnings related to line lengths being >80 characters